frontend/language/src/nb.json

```diff
@@ -638,14 +638,16 @@
   "schema_editor.integer": "Heltall",
   "schema_editor.keyword": "Nøkkelord",
   "schema_editor.loading": "Laster",
+  "schema_editor.maximum_inclusive": "Større enn",
+  "schema_editor.maximum_exclusive": "Mindre enn eller lik",
+  "schema_editor.minimum_inclusive": "Mindre enn",
+  "schema_editor.minimum_exclusive": "Større enn eller lik",
```

Comment:
Being extremely nitpicky here 🙈 But these two that start with `minimum` should be just above `schema_editor.minProperties` to be in alphabetical order.

Reply:
Small things mean alot in programming world, you are right , I just moved alle lines and forget to think alphabetically for each one.